content/millennium/r4/encounters/encounter.md

```diff
@@ -0,0 +1,99 @@
+---
+title: Encounter | R4 API
+---
+
+# Encounter
+
+* TOC
+{:toc}
+
+## Overview
+
+Placeholder
+
+## Search
+
+Search for Encounters that meet supplied query parameters:
+
+    GET /Encounter?:parameters
+
+_Implementation Notes_
+
+* The [Encounter.hospitalization.destination] will be returned as a reference to a [contained] location resource.
+* The [Encounter.location.location] may be returned as a reference to a [contained] location resource.
+
+### Authorization Types
+
+<%= authorization_types(practitioner: true, patient: false, system: true)%>
+
+### Parameters
+
+ Name       | Required?                  | Type          | Description
+------------|----------------------------|---------------|-------------------------------------------------------------------------------------------------------
+ `_id`      | This or patient or subject | [`token`]     | The logical resource id associated with the Encounter. Example: `7891`
+ `patient`  | This or subject or _id     | [`reference`] | The patient present at the encounter. Example: `12345`
+ `subject`  | This or patient or _id     | [`reference`] | The patient present at the encounter. Example: subject=Patient/1316024 or subject:Patient=1316024
+ [`_count`] | No                          | [`number`]    | The maximum number of results to return.
+
+### Headers
+
+ <%= headers %>
```

Comment:
use r4 headers (i.e. fhir+json)

Reply:
I've added the r4 default headers as an option: https://github.com/cerner/fhir.cerner.com/pull/238/commits/761f781e1eb58169b9b02d2530719b468b68a24d#diff-6590262e197180cc09f6ad70a2a01a5bL38

https://github.com/cerner/fhir.cerner.com/pull/238/commits/761f781e1eb58169b9b02d2530719b468b68a24d#diff-7718c11ed130ac00a9854e0b089f74b4L27

(fhir_json may not be a very good parameter name, taking suggestions)

Reply:
seems fine for now but maybe something relating to r4 is a good name for a future change